doc/containers.md

```diff
@@ -86,20 +86,22 @@ user.\*                                 | string    | -                 | n/a
-volatile.base\_image            | string    | -             | The hash of the image the container was created from, if any.
-volatile.idmap.base             | integer   | -             | The first id in the container's primary idmap range
-volatile.idmap.current          | string    | -             | The idmap currently in use by the container
-volatile.idmap.next             | string    | -             | The idmap to use next time the container starts
-volatile.last\_state.idmap      | string    | -             | Serialized container uid/gid map
-volatile.last\_state.power      | string    | -             | Container state as of last host shutdown
-volatile.\<name\>.host\_name    | string    | -             | Network device name on the host (for nictype=bridged or nictype=p2p, or nictype=sriov)
-volatile.\<name\>.hwaddr        | string    | -             | Network device MAC address (when no hwaddr property is set on the device itself)
-volatile.\<name\>.name          | string    | -             | Network device name (when no name propery is set on the device itself)
-
+Key                                        | Type      | Default       | Description
+:--                                        | :---      | :------       | :----------
+volatile.apply\_quota                      | string    | -             | Disk quota to be applied on next container start
+volatile.apply\_template                   | string    | -             | The name of a template hook which should be triggered upon next startup
+volatile.base\_image                       | string    | -             | The hash of the image the container was created from, if any.
+volatile.idmap.base                        | integer   | -             | The first id in the container's primary idmap range
+volatile.idmap.current                     | string    | -             | The idmap currently in use by the container
+volatile.idmap.next                        | string    | -             | The idmap to use next time the container starts
+volatile.last\_state.idmap                 | string    | -             | Serialized container uid/gid map
+volatile.last\_state.power                 | string    | -             | Container state as of last host shutdown
+volatile.\<name\>.host\_name               | string    | -             | Network device name on the host (for nictype=bridged or nictype=p2p, or nictype=sriov)
+volatile.\<name\>.hwaddr                   | string    | -             | Network device MAC address (when no hwaddr property is set on the device itself)
+volatile.\<name\>.name                     | string    | -             | Network device name (when no name propery is set on the device itself)
+volatile.\<name\>.last.ip.routes.host_name | string    | -             | Network device name used for host side static routes (for nictype=bridged or nictype=p2p)
```

Comment:
@stgraber would you prefer these keys to be:

```
volatile.\<name\>.last.routes.host_name
volatile.\<name\>.last.routes.ipv4
volatile.\<name\>.last.routes.ipv6
```

So that they are all in the same hierarchy?

Reply:
I've changed them to:

```
volatile.\<name\>.last.routes.host_name
volatile.\<name\>.last.ipv4.routes
volatile.\<name\>.last.ipv6.routes
```